.golangci.yml

```diff
@@ -1,9 +1,104 @@
-issues:
-  exclude-rules:
-    - text: "SA1019: package github.com/golang/protobuf/proto is deprecated: Use the \"google.golang.org/protobuf/proto\" package instead."
-      linters:
-        - staticcheck
-    - text: "SA1019: clientCtx.JSONCodec is deprecated: Codec codec will be changed to Codec: codec.Codec"
-      linters:
-        - staticcheck
+run:
+  timeout: 5m
+  disable-all: true
+  enable:
```

Comment:
How did you decide on this list of lint checks to run?

Reply:
Well I have written that list on many many chains in Cosmos and it seems to be approximately the correct ones

Reply:
Oh hey I'm sorry this is probably too long of a list

Reply:
Gotcha, this makes sense